src/pages/index.html

```diff
@@ -1,29 +1,332 @@
-{% extends "layout.html" %}
+{% extends "layout.html" %} {% set title = 'Home' %} {% block content %}
 
-{% set title = 'Home' %}
+<section>
+  <div class="header-container overflow-hidden container-fluid">
+    <div class="container">
+      <div class="d-flex flex-column align-items-center">
+        <h1
+          class="header-container--title d-flex flex-column align-items-center"
+        >
+          <span class="animated swing">
+            <span class="header-container--title--marker"
+              >{{headerDataEn.title.part1.marker}}</span
+            >
+            {{headerDataEn.title.part1.main}}
+          </span>
+          <span class="animated bounceInRight">
+            {{headerDataEn.title.part2.main}}
+            <span class="header-container--title--marker"
+              >{{headerDataEn.title.part2.marker}}</span
+            >
+          </span>
+        </h1>
 
-{% block content %}
+        <p class="header-container--info animated fadeIn">
+          {{headerDataEn.info}}
+        </p>
+        <a
+          href="{{headerDataEn.btn.link}}"
+          class="btn header-container--btn delay animated fadeIn"
+        >
+          {{headerDataEn.btn.name}}
+        </a>
+      </div>
+      <div class="row justify-content-center animated delay fadeIn">
+        <div class="col-sm-10 col-12">
+          <div class="header-container--dashboard-box">
+            <img src="../../assets/images/main/dashboard.png" alt="dashboard" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
 
-    <div class="container">
-        <div class="row">
-            <div class="col-sm-3">
-                {% include 'component/card.html' %}
-                {{ cards.mainCard({type: 'text', name: 'name'}) }}
-                {{ cards.mainCard({type: 'number', name: 'phone'}) }}
+<section>
+  <div class="container">
+    <div class="users-container d-flex flex-column">
+      <h1 class="users-container--title">
+        {{ homeDataEn.users.title.main }}
+        <span class="users-container--title--marker animated delay fadeIn">
+          {{ homeDataEn.users.title.marker }} </span
+        >.
+      </h1>
+      <div
+        class="users-container--list d-flex justify-content-center align-items-center flex-wrap"
+      >
+        {% for i in range(1, 6) %}
+        <img src="../assets/images/users/user-{{ i }}.png" alt="user-logo" />
+        {% endfor %}
```

Comment:
Missing logos

Reply:
i think it works well, please check it again
![logos](https://github.com/user-attachments/assets/9127f07a-2112-4d8f-a732-27dc1af2efad)